synapseclient/evaluation.py

```diff
@@ -195,12 +222,33 @@ def getURI(cls, id):
         return '/evaluation/submission/%s/status' % id
 
     def __init__(self, **kwargs):
+        annotations = kwargs.get('submissionAnnotations', {})
```

Comment:
Since the `__init__` will most likely be filled in by the rest call, it will definitely have synapse style annotations.  Unsure if the conversion is necessary here?

Reply:
I think it would be nice if the the entire time the object was in the client it was expected to be of a certain shape, in this case the Annotations typed dictionary, so I think it's useful to convert it on the deserialization from the backend.